android/app/src/main/java/chat/rocket/reactnative/Ejson.java

```diff
@@ -0,0 +1,27 @@
+package chat.rocket.reactnative;
+
+public class Ejson {
+    String host;
+    String rid;
+    Sender sender;
+
+    public String getAvatarUri() {
+        return this.removeTrailingSlash(this.host) + "/avatar/" + this.sender.username;
```

Comment:
If `Accounts_AvatarBlockUnauthenticatedAccess` is enabled, avatars won't load.
We should add credentials to this url.